.circleci/config.yml

```diff
@@ -341,7 +341,6 @@ jobs:
           keys:
             - influxdb-gomod-{{ checksum "go.sum" }} # Matches based on go.sum checksum.
       - run: ./scripts/ci/lint/flags.bash
-      - skip_if_not_master
       - save_cache:
```

Comment:
I don't think we want to save the module or build cache in this job, or in the `golint` job for that matter. It should only be enabled on the `gotest` job.

The build cache should include the test cache, and if the lint jobs save the cache first (which is likely since linting should happen much faster than running tests), then the cache we save will not necessarily contain a warm cache for tests, and subsequent runs will take more time to run the tests from a cold cache.

Reply:
Thanks for the explanation! I'v removed the caching for both lint jobs.